vignettes/using-based-on.Rmd

```diff
@@ -63,7 +63,10 @@ if (Sys.which("bbi") == "") {
 The modeling process will always start with an initial model, which we create with the `new_model()` call.
 
 ```{r first model}
-mod1 <- new_model(.yaml_path = "1.yaml", .description = "one compartment base model")
+mod1 <- new_model(
+  system.file("nonmem/1", package = "rbabylon"), 
```

Comment:
same as above

Reply:
We can resolve this by passing `.path = "../inst/nonmem/1"` rather than calling `system.file()`.